test/data/ln/node-defaults.yaml

```diff
@@ -1,8 +1,40 @@
+# Enable caddy, collectLogs and metricsExport to activate publish lnd-exporter metrics
+caddy:
+  enabled: false
+
+#Core configs
 image:
   repository: bitcoindevproject/bitcoin
   pullPolicy: IfNotPresent
   tag: "27.0"
+collectLogs: false
+metricsExport: false
 
+#LN configs
+ln:
+  lnd: true
 lnd:
   defaultConfig: |
-    color=#000000
\ No newline at end of file
+    color=#000000
+  config: |
+    bitcoin.timelockdelta=33
+  metricsExport: false
+  prometheusMetricsPort: 9332
+  extraContainers:
+    - name: lnd-exporter
+      image: bitdonkey/lnd-exporter:0.1.2
+      imagePullPolicy: IfNotPresent
+      env:
+        - name: ADMIN_MACAROON_HEX
+          value: 0201036c6e6402f801030a1062beabbf2a614b112128afa0c0b4fdd61201301a160a0761646472657373120472656164120577726974651a130a04696e666f120472656164120577726974651a170a08696e766f69636573120472656164120577726974651a210a086d616361726f6f6e120867656e6572617465120472656164120577726974651a160a076d657373616765120472656164120577726974651a170a086f6666636861696e120472656164120577726974651a160a076f6e636861696e120472656164120577726974651a140a057065657273120472656164120577726974651a180a067369676e6572120867656e657261746512047265616400000620b17be53e367290871681055d0de15587f6d1cd47d1248fe2662ae27f62cfbdc6
+        - name: METRICS
+          value: >
+            lnd_balance_channels=parse("/v1/balance/channels","balance") 
+            lnd_local_balance_channels=parse("/v1/balance/channels","local_balance.sat") 
+            lnd_remote_balance_channels=parse("/v1/balance/channels","remote_balance.sat") 
+            lnd_block_height=parse("/v1/getinfo","block_height") 
+            lnd_peers=parse("/v1/getinfo","num_peers")
```

Comment:
This is really cool but I don't see a test for it. I think that's ok although I would like to at least have a grafana dashboard I can easily load to see these data. Could be a follow up PR